// Copyright (c) 2020-2021 The Open-Transactions developers
// This Source Code Form is subject to the terms of the Mozilla Public
// License, v. 2.0. If a copy of the MPL was not distributed with this
// file, You can obtain one at http://mozilla.org/MPL/2.0/.

syntax = "proto2";

package opentxs.proto;
option java_package = "org.opentransactions.proto";
option java_outer_classname = "OTPeerRequestHistory";
option optimize_for = LITE_RUNTIME;

import public "PeerEnums.proto";
import public "PeerRequestWorkflow.proto";

message PeerRequestHistory
{
    optional uint32 version = 1;
    optional PeerRequestType type = 2;
    repeated PeerRequestWorkflow workflow = 3;
}
